src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>To Do List</title>
+</head>
+
+<body>
+    <section class="container">
+        <div class="todo-headline">
+            <h1>Today's To Do</h1>
+            <button type="button"><i class="fa fa-refresh"></i></button>
+        </div>
+        <input class="taskList" type="text" name="taskList" id="taskList" placeholder="Add to your list...">
+        <ul class="todo-container">
+            <li class="todo-item">
+                <div class="todo-content">
+                    <input type="checkbox" name="checkbox" value = "0"/>
+                    <p>wash the dishes</p>
+                </div>
+                <button type="button"><i class="fa fa-ellipsis-v"></i></button>
+            </li>
+            <li class="todo-item">
+                <div class="todo-content">
+                    <input type="checkbox" name="checkbox" value = "1" />
+                    <p>complete To Do list project</p>
+                </div>
+                <button type="button"><i class="fa fa-ellipsis-v"></i></button>
+            </li>
```

Comment:
## OPTIONAL

- [ ] Kindly remove hard coded values in your index.html file as this is not required, your todos displayed should come from the array of objects in index.js as you have rightly done 👍🏿